app/controllers/benchmark_runs_controller.rb

```diff
@@ -31,6 +31,10 @@ def create
     benchmark_run.result = params[:benchmark_run][:result]
     benchmark_run.save!
 
+    if params[:commit_hash]
+      BenchmarkRegressionJob.perform_later(benchmark_run, initiator, benchmark_type, benchmark_result_type, benchmark_type_params[:category])
```

Comment:
It is not how many parameters that is the important bit. It is what you pass in :wink: